etc/config-template.toml

```diff
@@ -245,8 +245,12 @@
 # set backup path, if not set, use "backup" under store path.
 # backup-dir = "/tmp/tikv/store/backup"
 
+# Allows OS to incrementally sync files to disk while they are being
+# written, asynchronously, in the background.
+# bytes-per-sync = "1MB"
```

Comment:
do we bench this?

Reply:
No, we haven't, but this is a default number in rocksdb if we set rate limiter, so I think 1MB is fine.